python/extensions.bzl

```diff
@@ -127,3 +152,59 @@ pip = module_extension(
         "parse": tag_class(attrs = _pip_parse_ext_attrs()),
     },
 )
+
+def _host_hub_impl(repo_ctx):
+    repo_ctx.file("BUILD.bazel", """\
```

Comment:
I don't think this BUILD file actually needs anything within it?

Reply:
Don't we need the export?

Reply:
No, because it's only used by repo operations, which don't understand the content within BUILD files.

Reply:
>  BUILD file not found in directory '' of external repository @rules_python~override~python~python3_host_interpreter. Add a BUILD file to a directory to mark it as a package.

So we need it :)